components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -1068,6 +1068,10 @@ + (CGFloat)floatingPlaceholderScaleDefault {
 
 + (void)setFloatingPlaceholderScaleDefault:(CGFloat)floatingPlaceholderScaleDefault {
   _floatingPlaceholderScaleDefault = floatingPlaceholderScaleDefault;
+  if (floatingPlaceholderScaleDefault <= 0) {
```

Comment:
Just curious, what was the situation where you found this was an issue?

Reply:
So for other properties we have null_resettable where we could set the property to null to reset it to default. but for this property that didn't exist and setting the value to zero or less is meaning less so I added the option. I also needed that incase we are applying a new font scheme which doesn't have enough info to set a new scale.

Reply:
Thinking ahead! Got it.